vapoursynth build error, can't find `VSScript4.h`

### Important Information

```text
- mpv version: git master
- Platform version: openSUSE Tumbleweed 20260610
- Meson version: 1.11.1
```

### Reproduction Steps

Install openSUSE Tumbleweed

git clone https://github.com/mpv-player/mpv-build.git
cd mpv-build
./rebuild -j6

### Expected Behavior

Successful build

### Actual Behavior

After a recent system update, mpv-build began to fail with the following output:

> FAILED: [code=1] libmpv.so.2.5.0.p/video_filter_vf_vapoursynth.c.o 
> cc -Ilibmpv.so.2.5.0.p -I. -I.. -I../include -Icommon -Ietc -Iplayer/javascript -Iplayer/lua -Isub -Ivideo/out -I/home/tofu/Software/mpv-build/build_libs/include -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/fribidi -I/usr/include/freetype2 -I/usr/include/libxml2 -I/usr/include/lua5.1 -I/usr/include/uchardet -I/usr/lib64/pkgconfig/../include -I/usr/include/pipewire-0.3 -I/usr/include/spa-0.2 -I/usr/include/libdrm -I/usr/include/libdisplay-info -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/glycin-2 -I/usr/include/libseccomp -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/wayland -I/usr/include/libxkbcommon -fvisibility=hidden -fdiagnostics-color=always -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_FILE_OFFSET_BITS=64 -Wdisabled-optimization -Wempty-body -Wformat -Wimplicit-fallthrough -Woverflow -Wparentheses -Wpointer-arith -Wshadow -Wundef -Wvla -Wno-cast-function-type -Wno-format-zero-length -Wno-missing-field-initializers -Wno-sign-compare -Wno-switch -Wno-unused-parameter -fno-math-errno -fno-signed-zeros -fno-trapping-math -Werror=format-security -Wno-unused-result -D_GNU_SOURCE -Wmissing-prototypes -Wstrict-prototypes -Werror=implicit-function-declaration -Wno-pointer-sign -DWINICONV_CONST= -fPIC -pthread -D_REENTRANT -fno-strict-aliasing -fno-strict-overflow -DPL_HAVE_PTHREAD -DPTHREAD_HAS_SETCLOCK -DPL_STATIC -MD -MQ libmpv.so.2.5.0.p/video_filter_vf_vapoursynth.c.o -MF libmpv.so.2.5.0.p/video_filter_vf_vapoursynth.c.o.d -o libmpv.so.2.5.0.p/video_filter_vf_vapoursynth.c.o -c ../video/filter/vf_vapoursynth.c
> ../video/filter/vf_vapoursynth.c:25:10: fatal error: VSScript4.h: No such file or directory
>    25 | #include <VSScript4.h>
>       |          ^~~~~~~~~~~~~
> compilation terminated.
> [289/355] Compiling C object libmpv.so.2.5.0.p/player_lua.c.o
> ninja: build stopped: subcommand failed.

/usr/include/vapoursynth/VSScript4.h exists so I'm not sure the cause of this error. This only applies to master, the release branch builds successfully.

### Log File

[meson-log.txt](https://github.com/user-attachments/files/28930759/meson-log.txt)

### I carefully read all instruction and confirm that I did the following:

- [x] I tested and confirmed that the issue exists with the [latest release version](https://github.com/mpv-player/mpv/releases/latest) or newer.
- [x] I provided all required information including system and mpv version.
- [x] I produced the log file with the exact same set of parameters, and conditions used in "Reproduction Steps".
- [x] I produced the log file while the behaviors described in "Actual Behavior" were actively observed.
- [x] I attached the full, untruncated log file.

--- a/meson.build
+++ b/meson.build
@@ -783,7 +783,10 @@ if features['uchardet']
     dependencies += uchardet
 endif
 
-vapoursynth = dependency('vapoursynth', version: '>= 56', required: get_option('vapoursynth'))
+vapoursynth = dependency('vapoursynth-script', version: '>= 56', required: false)
+if not vapoursynth.found()
+    vapoursynth = dependency('vapoursynth', version: '>= 56', required: get_option('vapoursynth'))
+endif
 features += {'vapoursynth': vapoursynth.found()}
 if features['vapoursynth']
     dependencies += vapoursynth.partial_dependency(compile_args: true, includes: true)

